src/types/options.ts

```diff
@@ -96,9 +101,14 @@ export interface WidgetOptions extends Partial<Pick<OktaAuthOptions,
   consent?: {
     cancel?: SimpleCallback;
   };
-  useInteractionCodeFlow?: boolean;
+  useClassicEngine?: boolean;
   hooks?: HooksOptions;
   proxyIdxResponse?: any;
+
+  /**
+   * @deprecated since version 7.0
```

Comment:
Awesome. Does it make sense to explain here or point to the `useClassicEngine` option in the README or migration guide?
If this shows up as an error/warning in VS Code or CI, it won't be very informative.
Changing the annotation can wait for 7.0.1